jest.config.js

```diff
@@ -1,4 +1,7 @@
 module.exports = {
   verbose: true,
-  setupFilesAfterEnv: ["./jest.setup.ts"],
+  testEnvironment: 'jsdom',
```

Comment:
Hei thanks for noticing this. Jest is not my strength.
I changed it to json because of the session using window. 
Example in the typeIndexLogic.test.ts I get:
 ```
 The error below may be caused by using the wrong test environment, see https://jestjs.io/docs/configuration#testenvironment-string.
    Consider using the "jsdom" test environment.
    ReferenceError: window is not defined
       6 | let authSession: Session
       7 | // @ts-ignore
    >  8 | if (!window.authSession) {
```
And funny enough adding the comment you mentioned above did not change the situation. 
This might go away if I move the authSession to solid.ui back... but if not.. not sure. 
@angelo-v what do you think?

Reply:
Not sure why it does not work with your suggestion. I found even the PR on jest for it: https://github.com/facebook/jest/pull/2859

Reply:
Did you put the comment on the top of the test file? It should work, I used it before.

Reply:
@angelo-v sorry to report that is still did not work out as expected. I tried with the comment on top and also under import and same result as mentioned above. I wonder what is wrong...

Reply:
Oops, sorry so my example was wrong :o

The messages you get indicate that promises did not finish before jest ended. I need to take a look at it in this case

Reply:
@theRealImy can you share some details: What test is ist and what node version are you using? Cannot reproduce it straight away